pkg/controllers/common/common.go

```diff
@@ -0,0 +1,112 @@
+/*
```

Comment:
Can we not call it _common_ please? At least the filename should match what it holds. Call it `managed_secret_client` or something please. :)